ee/session_recordings/test/test_session_recording_extensions.py

```diff
@@ -100,14 +93,21 @@ def test_persists_recording_from_blob_ingested_storage(self):
 
                 # this recording already has several files stored from Mr. Blobby
                 # these need to be written before creating the recording object
+                blob_path = f"{TEST_BUCKET}/team_id/{self.team.pk}/session_id/{session_id}/data"
                 for file in ["a", "b", "c"]:
-                    blob_path = f"{TEST_BUCKET}/team_id/{self.team.pk}/session_id/{session_id}/data"
                     file_name = f"{blob_path}/{file}"
                     write(file_name, f"my content-{file}".encode())
 
+                assert object_storage_client().list_objects(OBJECT_STORAGE_BUCKET, blob_path) == [
+                    f"{blob_path}/a",
+                    f"{blob_path}/b",
+                    f"{blob_path}/c",
+                ]
```

Comment:
my change at first broke this test but it failed elsewhere... added this assertion so that i got a meaningful failure